_posts/2015-01-15-sandstorm-1.3M-seed-round-pay-it-forward.md

```diff
@@ -41,9 +41,9 @@ For those who missed the Indiegogo campaign, [we are now accepting pre-orders fo
 
 ### Get Involved
 
-We regularly post internal meeting notes and answer questions on [the sandstorm-dev mailing list](https://groups.google.com/group/sandstorm-dev), and we are usually available to chat on [IRC (#sandstorm on freenode)](https://kiwiirc.com/client/irc.freenode.net/?channel=#sandstorm). Feel free to come talk to us in either place!
+We regularly post internal meeting notes and answer questions on [the sandstorm mailing list](https://groups.io/g/sandstorm-user-group), and we are usually available to chat on [IRC (#sandstorm on freenode)](https://kiwiirc.com/client/irc.freenode.net/?channel=#sandstorm). Feel free to come talk to us in either place!
```

Comment:
You probably thought of this, but maybe we should have another PR to change freenode links to Matrix.

Reply:
Why'd you remove the `-dev` from the text here and not the one immediately below?

Reply:
Nevermind I see

Reply:
I thought about revising history on IRC/Matrix, but one of the reasons I have hesitated is not having a good answer on what to do with it yet since our Matrix hasn't gotten fixed yet.